docs/07-module-block-7/01-duration-terms.adoc

```diff
@@ -0,0 +1,77 @@
+// tag::DE[]
+|===
+| Dauer: frei | Übungszeit: frei
+|===
+
+Rund um DDD gibt es viele Themen, die einen Bezug dazu haben und vom Trainer nach Bedarf auch gelehrt werden können.
```

Comment:
Lehrpläne verzichten in der Regel auf die rein männliche Form von Wörtern, also sollte hier
- von/vom Trainer:in
- von der Trainerin oder vom Trainer
- von den Trainierenden 
- von Trainer:innen

gewählt werden.